edi/models/edi_exchange_record.py

```diff
@@ -151,6 +151,8 @@ def needs_ack(self):
 
     @property
     def record(self):
+        if not self.model or self.model not in self.env:
```

Comment:
The same change applied on 12.0

Reply:
I think we fixed this on https://github.com/OCA/edi/pull/346 CC @acsonefho

Reply:
Yep, Cherry picked the commit, as the PR is still on draft and this solves some issues :wink: